src/codeu/chat/codeU_db/ResetDatabase.java

```diff
@@ -0,0 +1,122 @@
+package codeu.chat.codeU_db;
+
+import java.sql.*;
+
+public class ResetDatabase
+{
+    public static void main( String args[] )
+    {
+        Connection c = null;
+        Statement stmt = null;
+
+        try {
+            Class.forName("org.sqlite.JDBC");
+            c = DriverManager.getConnection("jdbc:sqlite:./bin/codeu/chat/codeU_db/ChatDatabase.db");
+            System.out.println("Opened database successfully");
+        } catch ( Exception e ) {
+            System.err.println( e.getClass().getName() + ": " + e.getMessage() );
+            System.exit(0);
+        }
+
+        //drop tables before creating them
+        try {
+            stmt = c.createStatement();
+            String sql = "DROP TABLE USERS";
+            stmt.executeUpdate(sql);
+            stmt.close();
+
+            stmt = c.createStatement();
+            sql = "DROP TABLE CONVERSATIONS";
+            stmt.executeUpdate(sql);
+            stmt.close();
+
+            stmt = c.createStatement();
+            sql = "DROP TABLE USER_CONVERSATION";
+            stmt.executeUpdate(sql);
+            stmt.close();
+
+            stmt = c.createStatement();
+            sql = "DROP TABLE MESSAGES";
+            stmt.executeUpdate(sql);
+            stmt.close();
+
+        }catch ( Exception e ) {
```

Comment:
ditto. A helper function that just took the connection and SQL string would  remove a lot of duplication in this file as well.

indentation and horizontal whitespace needs to be fixed here.